chat-plugins/trivia.js

```diff
@@ -1005,7 +1005,10 @@ const commands = {
 	submit: 'add',
 	add: function (target, room, user, connection, cmd) {
 		if (room.id !== 'questionworkshop') return this.errorReply('This command can only be used in Question Workshop.');
-		if ((cmd === 'add' && !this.can('mute', null, room)) || (cmd === 'submit' && !this.can('broadcast', null, room)) || !target) return false;
+		if ((cmd === 'add' && !this.can('mute', null, room)) ||
+            (cmd === 'submit' && !this.can('broadcast', null, room)) ||
```

Comment:
Use a couple tabs, same way the rest of PS does
Like this:
```javascript
     if ((cmd === 'add' && !this.can('mute', null, room)) ||
            (cmd === 'submit' && !this.can('broadcast', null, room)) ||
            !target) {
         return false;
     }
````

Reply:
huh that's odd on my editor I lined it up so it looks nice but I can clearly see that it isn't here

Reply:
OK I think that looks correct now